src/main/io/osd.c

```diff
@@ -600,6 +599,20 @@ static bool osdDrawSingleElement(uint8_t item)
 
             break;
         }
+		
+    case OSD_MOTOR_DIAG:
+        if( areMotorsRunning() ) {
```

Comment:
Please use spaces on the outside of parens, and not on the inside, i.e. `if (areMotorsRunning()) {`.